index.js

```diff
@@ -1,13 +1,111 @@
-const express = require('express');
-const helmet = require('helmet');
+const express = require("express");
+const helmet = require("helmet");
+
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
+  useNullAsDefault: true, //important
+  connection: {
+    filename: "./data/lambda.sqlite3"
+  }
+};
+
+const db = knex(knexConfig);
 
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
-
 // endpoints here
 
+server.post("/api/zoos", (req, res) => {
+  db("zoos")
+    .insert(req.body)
```

Comment:
It would still be good to have conditionals to test if you have the right things on the req.body.